src/controller/student/student.service.ts

```diff
@@ -225,8 +227,11 @@ export class StudentService {
         }
         return acc;
       }, {upcoming: [], ongoing: [] });
-
-      return [null, {status:helperVariable.success,code: STATUS_CODES.OK,filterClasses,totalClasses,totalPages : !isNaN(limit) ? Math.ceil(totalClasses/limit) : 1, message: 'Upcoming classes retrieved successfully'}];
+      if(Number(totalClasses) == 0)
+        {
+          return [null, {status:helperVariable.success,code: STATUS_CODES.OK,message: 'No upcoming classes'}];
+        }      
```

Comment:
If you’re returning anything beyond the status code and message, expect it to be included in the data section.
{
    "statusCode": 200,
    "message": "Resource created successfully",
    "data": {
        "userId": 123,
        "username": "john_doe",
        "email": "john@example.com"
    }
}